### PR TITLE
miniDREAM RStudio env test instance

### PR DESCRIPTION
**Pull request by milen-sage**

### CODE REVIEW

File: config/prod/minidream-R-env-test-instance-ec2.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Provision an EC2 instance
+template_path: templates/managed-ec2-v2.yaml
```
Comment (zaro0508) [inline config/prod/minidream-R-env-test-instance-ec2.yaml]: there is new version, please change to `templates/managed-ec2-v4.yaml`

Comment (milen-sage) [reply config/prod/minidream-R-env-test-instance-ec2.yaml]: Updated versions.

File: config/prod/minidream-R-env-test-instance-ec2.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  Project: "CSBC"
+  # The resource owner
+  OwnerEmail: "milen.nikolov@sagebase.org"
+  # EC2 instance type (available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.nano"
+   # Integration with our jumpcloud directory service (do not change)
+  JcConnectKey: !ssm /infra/JcConnectKey
+  JcServiceApiKey: !ssm /infra/JcServiceApiKey
+  JcSystemsGroupId: !ssm /infra/JcSystemsGroupId
+hooks:
```
Comment (zaro0508) [inline config/prod/minidream-R-env-test-instance-ec2.yaml]: sorry, didn't notice this on the 1st review.  this hooks section should be
```
hooks:
  after_create:
    - !ec2_notify {{stack_group_config.aws_account_name}} {{stack_group_config.aws_account_email}}

```

Comment (milen-sage) [reply config/prod/minidream-R-env-test-instance-ec2.yaml]: Updated hooks config.

**Pull request merged**